Python/ast.c

```diff
@@ -2705,14 +2705,14 @@ ast_for_expr(struct compiling *c, const node *n)
 }
 
 static expr_ty
-ast_for_call(struct compiling *c, const node *n, expr_ty func)
+ast_for_call(struct compiling *c, const node *n, expr_ty func, int allowgen)
```

Comment:
`allowgen` should be `bool` parameter and the callers should pass `true` or `false`.

Reply:
```
Python/ast.c:614:72: error: unknown type name ‘bool’; did you mean ‘_Bool’?
Python/ast.c:1548:53: error: ‘false’ undeclared (first use in this function); did you mean ‘fabsl’?
Python/ast.c:2371:60: error: ‘true’ undeclared (first use in this function); did you mean ‘time’?
```

Reply:
You need to `#include <stdbool.h>`